templates/settings/apps.html

```diff
@@ -0,0 +1,33 @@
+{% extends "settings/base.html" %}
+
+{% block subtitle %}Apps{% endblock %}
+
+{% block content %}
+    <h2>Apps</h2>
+    <div>
+        {% for token in object_list %}
+            <p>
+                <h3>{{token.application.name}}</h3>
+                <table>
```

Comment:
There's existing setting page table styling, please use that! It'll allow for addition of "revoke token" actions and the like later.

Reply:
Do you mean the "items" CSS class used in import_export.html?

Reply:
> There's existing setting page table styling, please use that! It'll allow for addition of "revoke token" actions and the like later.

Should this "revoke token" button be added as a part of this PR?